server/routerlicious/packages/services-shared/src/healthCheckEndpoints.ts

```diff
@@ -0,0 +1,129 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { RequestHandler, Router, Request, Response, NextFunction } from "express";
+import {
+	throttle,
+	IThrottleMiddlewareOptions,
+	getParam,
+} from "@fluidframework/server-services-utils";
+import * as core from "@fluidframework/server-services-core";
+import { StartupChecker } from "./startupChecker";
+import { LumberEventName, Lumberjack } from "@fluidframework/server-services-telemetry";
+
+export interface IReadinessCheck {
```

Comment:
Please add doc comments to the overall interface and its member properties.

For example,
```ts
/**
 * Checks if a service or functionality is ready for use.
 * @internal
 */
export interface IReadinessCheck {
  /**
   * Whether the service/functionality is ready for use.
   */
  isReady(): Promise<boolean>;
}
```